src/workarounds/gpu_arrays.jl

```diff
@@ -0,0 +1,16 @@
+#TODO: remove this when it is implemented in GPUArrays
+import LinearAlgebra.dot, LinearAlgebra.eigen, LinearAlgebra.RealHermSymComplexHerm
+using LinearAlgebra
+using GPUArrays
+using CUDA
+
```

Comment:
You should in each case add a comment with the issue numbers in GpuArrays, CUDA etc. where you track these missing functionalities.